content/en/docs/setup/production-environment/tools/kubeadm/kubelet-integration.md

```diff
@@ -161,6 +161,12 @@ Kubeadm deletes the `/etc/kubernetes/bootstrap-kubelet.conf` file after completi
 `kubeadm` ships with configuration for how systemd should run the kubelet.
 Note that the kubeadm CLI command never touches this drop-in file.
 
+{{< note >}}
+The contents below are just an example. If you don't want to use a package manager
+follow the guide outlined in the [Without a package manager](/docs/setup/production-environment/tools/kubeadm/install-kubeadm/#k8s-install-2))
+section.
+{{< /note >}}
+
```

Comment:
please move this above the actual example:

```
none
[Service]
```

Reply:
I wanted to do the same but there was an attribute ```none ,so it has to be above it?

Reply:
the note should be outside the "none" example block - above it.